gcp-function-http/src/main/java/io/micronaut/gcp/function/http/jackson/ObjectMapperCustomizer.java

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.gcp.function.http.jackson;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
-import io.micronaut.context.event.BeanCreatedEvent;
-import io.micronaut.context.event.BeanCreatedEventListener;
-
-import jakarta.inject.Singleton;
-
-/**
- * Customizes the jackson object mapper to ensure essential modules are registered.
- *
- * @author graemerocher
- * @since 1.2.0
- */
-@Singleton
```

Comment:
should we keep this but annotate with `@Requires(classes = JacksonConfiguration.class)` and add micronaut.jackson as `compileOnly`?